src/participant.rs

```diff
@@ -262,6 +262,7 @@ impl Identity {
 }
 
 // Need to implement `Hash` manually because `Signature` does not implement it
+#[allow(clippy::derive_hash_xor_eq)]
```

Comment:
@andiflabs I added this because of this lint error: https://rust-lang.github.io/rust-clippy/master/index.html#/derive_hash_xor_eq

the alternative is to compare hashes of identities instead of using the PartialEq trait. Lmk what you think